packages/mermaid/src/diagrams/git/gitGraphRenderer.js

```diff
@@ -523,7 +522,7 @@ export const draw = function (txt, id, ver, diagObj) {
   drawCommits(diagram, allCommitsDict, true);
 
   // Setup the view box and size of the svg element
-  setupGraphViewbox(undefined, diagram, gitGraphConfig.diagramPadding, conf.useMaxWidth);
+  setupGraphViewbox(undefined, diagram, gitGraphConfig.diagramPadding, gitGraphConfig.useMaxWidth);
```

Comment:
I feel both values should be considered. 

```suggestion
  setupGraphViewbox(undefined, diagram, gitGraphConfig.diagramPadding, gitGraphConfig.useMaxWidth ?? conf.useMaxWidth);
```

Reply:
@knsv @sidharthv96 

Thanks for the review.
Fixed to use global settings.